src/main/java/org/jabref/logic/importer/fileformat/IsiImporter.java

```diff
@@ -32,11 +32,10 @@
  * Importer for the ISI Web of Science, INSPEC and Medline format.
  * <p>
  * Documentation about ISI WOS format:
- * <p>
+ * </p>
  * <ul>
```

Comment:
Line 34 is a heading. Thus closing the paragraph is not right.

Make Line 32 a paragraph and start a new one for the documentation. Finish the paragraph at the end.

Full "solution":

```java
/**
 * <p>
 * Importer for the ISI Web of Science, INSPEC and Medline format.
 * </p>
 * <p>
 * Documentation about ISI WOS format:
 * <ul>
 * 
 * <li>https://web.archive.org/web/20131031052339/http://wos.isitrial.com/help/helpprn.html</li>
 * </ul>
 * <p>
 * <ul>
 * <li>Deal with capitalization correctly</li>
 * </ul>
 * </p>
 */